src/ArticleSection.jsx

```diff
@@ -0,0 +1,23 @@
+import ArticleList from "./ArticleList";
+import ArticleVotes from "./ArticleVotes";
+import { useState, useEffect } from "react";
+import axios from "axios";
+const ArticleSection = () => {
+  const [articles, setArticles] = useState([]);
+  useEffect(() => {
+    axios
```

Comment:
You could separate out your api calls into a separate api.js file which you import and call here, and throughout your app when you make API calls.